app/connectors/IdentityMatchConnector.scala

```diff
@@ -58,7 +59,7 @@ class IdentityMatchConnector @Inject()(val http: HttpClient, val appConfig: AppC
 
     implicit val hc: HeaderCarrier = HeaderCarrier(extraHeaders = headers(correlationId))
 
-    logger.info(s"Matching individual for correlationId: $correlationId")
+    logger.info(s"[Session ID: ${Session.id(hc)}] Matching individual for correlationId: $correlationId")
```

Comment:
@BriWak Not sure if we can confirm here but I suspect the SessionID will always be `None` here as we construct a new `hc` value above?